docs/application/dotnet/guides/app-management/data-control.md

```diff
@@ -34,51 +34,58 @@ The main features of the Tizen.Applications.DataControl namespace include:
 
 The data control use cases involve 2 applications. Each application plays a different role: one as the consumer, the other as the provider.
```

Comment:
The data control use cases involve two applications. Each application plays a different role: one as the consumer, the other as the provider.

Reply:
I changed 2 to two. Also I think it should be as **a** consumer, not as **the** consumer.

Reply:
@MathewP1, valid point, agree please make that change as well, for both 

....a consumer, the other as a provider.